config/code_schools.yml

```diff
@@ -11,6 +11,7 @@
 #  full_time:  Is your school full time? [true|false]
 #  hardware_included: Does tuition include hardware? [true|false]
 #  notes: <Addtional Comments> [optional]
+#  img: Image file name of logo
```

Comment:
Can you add a bit more info to this help text? We get a lot of outside contributors adding their schools so this should be dead simple to update.

Where do I put the image? What size does it have to be? What format?